frontend/src/views/Worksheets/WorksheetView.js

```diff
@@ -32,7 +32,7 @@ import listEnvironmentGroups from '../../api/Environment/listEnvironmentGroups';
 import { SET_ERROR } from '../../store/errorReducer';
 import listDatasetsOwnedByEnvGroup from '../../api/Environment/listDatasetsOwnedByEnvGroup';
 import listDatasetTables from '../../api/Dataset/listDatasetTables';
-import getSharedDatasetTables from '../../api/Dataset/getSharedDatasetTables';
+import getSharedDatasetTables from '../../api/DatasetTable/getSharedDatasetTables';
 import listDatasetTableColumns from '../../api/DatasetTable/listDatasetTableColumns';
 import searchEnvironmentDataItems from '../../api/Environment/listDatasetsPublishedInEnvironment';
```

Comment:
Currently, if I share N Tables in 1 dataset with a Consumer Group, the glue database name shows up N times if the consumer is querying the data using Worksheets feature.

I believe we use this `searchEnvironmentDataItems` api to query and set shared databases --> may just need to add a new query that pulls queries the same with a `.distinct()` filter on database name.

Reply:
I am going to fix this in a separate issue as it is a generic bug. Also from what I see in worksheets we are not using `paginated_shared_with_environment_group_datasets` and we are querying all environment databases.